src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -982,20 +985,25 @@ export class TerminalService extends Disposable implements ITerminalService {
 		}
 
 		let config = options?.config;
+		this._logService.debug(`#terminalService#createTerminal: config ${config}`);
 		if (!config && isLocalInRemoteTerminal) {
+			this._logService.debug(`#terminalService#createTerminal: !config and isLocalInRemoteTerminal`);
 			const backend = await this._terminalInstanceService.getBackend(undefined);
```

Comment:
Because of https://github.com/microsoft/vscode/pull/277920/commits/1b762fc6ca5743b863267aa24fdbf9d2cdcff0af#r2540901561 Code never reaches here actually.